rebasehelper/specfile.py

```diff
@@ -331,7 +331,7 @@ def get_not_used_patches(self):
         """
         return self.patches['not_applied']
 
-    def _comment_out_patches(self, patch_num, note=None):
+    def _comment_out_patches(self, patch_num, note=None, remove_patches=False):
```

Comment:
And it is necessary for commenting out inapplicable patches if `--disable-inapplicable-patches` option is set.